src/sqlancer/questdb/gen/QuestDBTableGenerator.java

```diff
@@ -27,13 +35,24 @@ public SQLQueryAdapter getQuery(QuestDBGlobalState globalState) {
             sb.append(" ");
             sb.append(columns.get(i).getType());
         }
-        sb.append(");");
+        sb.append(")");
+        // test index at Create Table - bug report already submitted
```

Comment:
Could you please adopt the pattern described at https://github.com/sqlancer/sqlancer/blob/master/CONTRIBUTING.md#unfixed-bugs to ignore unfixed bugs?

Reply:
fixed